### ISSUE TITLE
Connector: MYSQL [Source/Destination]

### ISSUE THREAD

**Issue by simonl2002** [ConduitIO/conduit#1553]
### Resource name

MYSQL

### Resource link

https://www.mysql.com/

### Connector type

Source

### Expected connector behavior

Connector should enable CDC stream of rows from mysql tables.

**Comment by simonl2002** [comment]
@nickchomey we've decided to create a dedicated MYSQl connector. Vitess uses different terminology and it just feels easier to to have dedicated connector.

**Comment by nadilas** [comment]
@nickchomey i understand the vitess connector is capable of handling a MySQL destination. Do you have an example setup? Can’t seem to connect from the connector to the MySQL instance (looking at the code suggests that it can only talk to a vitess vtable).

**Comment by simonl2002** [comment]
@nadilas the MySQL connector is being worked on here: https://github.com/conduitio-labs/conduit-connector-mysql

**Comment by nadilas** [comment]
Thanks @simonl2002 @nickchomey, that connector does not have an implementation for a destination yet. I was hoping to get a temporary solution until the destination MySQL becomes ready.

**Comment by nadilas** [comment]
I was actually toying with the idea of doing the destination code ourselves, but couldn’t yet put an effort estimate to the idea to weigh the feasibility. Thanks for the tip, I’ll look into the branch. 👍

**Comment by hariso** [comment]
The destination PR was reviewed, so I think it should be merged relatively soon. The Conduit team will be focusing on the MySQL connector (and a few others) a bit more now that we're done with the 0.12 release.:)

**Comment by hariso** [comment]
> I was actually toying with the idea of doing the destination code ourselves, but couldn’t yet put an effort estimate to the idea to weigh the feasibility. Thanks for the tip, I’ll look into the branch. 👍

Regarding doing the destination code yourselves: the Connector SDK should make it fairly simple to do so. The destination are simpler compared to source obviously and usually it's only batching that might require some additional though (Conduit makes batching possible, but doesn't and can't require connectors to do so). Just for the record, here are a few guidelines about writing connectors: https://github.com/ConduitIO/conduit/blob/4b66347eb89a2bb8cf641565f7602bf36bc33137/docs/writing-a-connector-guidelines.md.

**Comment by nadilas** [comment]
We just tested it today and seems to work okay. We need to get some processors in to convert e.g sql server datetime string to MySQL format, but the basics seem to work. 
Will do some further testing tomorrow, thanks guys

**Comment by lovromazgon** [comment]
@nadilas you're welcome to join our [Discord server](https://discord.meroxa.com/) (if you haven't already) and let us know how it goes or ask if something is unclear, we're happy to assist you 😉

**Comment by hariso** [comment]
FYI, the destination PR was merged.

**Comment by raulb** [comment]
This is done already. Will be released today.

**Comment by raulb** [comment]
@nickchomey Makes sense. We might release 0.1.0 today, but we'll make sure both #51 and #54 are released soon. #51 only needs a CI fix, and I just pinged @lovromazgon and @alarbada to follow up with you on #54 (it's going to need a rebase for now).
